.travis.yml

```diff
@@ -3,14 +3,11 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
+  - 1.0
+  - 1
```

Comment:
what's the distinction between 1.0 and 1?

Reply:
1 in this case means the latest point release, so 1.3 right now.